addons/sys_core/fnc_remoteStartSpeaking.sqf

```diff
@@ -143,9 +143,9 @@ private _result = false;
                 WARNING_1("Got start speaking event with non-existent radio id: %1",_radioId);
             };
         } else {
-            if (_onRadio == 5) then { // God Mode / Zeus
+            if (_onRadio == 5) then { // God Mode
```

Comment:
5 is a magic  number. I would define it so we can reuse it/adapt it in the future.

```suggestion
            if (_onRadio == REMOTE_SPEAKING_GOD_MODE) then { // God Mode
```

Reply:
Other values here would also have to get added, not sure this fits this PR scope.